eng/common/scripts/Test-SampleMetadata.ps1

```diff
@@ -221,7 +221,7 @@ begin {
         "azure-kubernetes-service",
         "azure-lab-services",
         "azure-language-understanding",
-        "azure-language-service",
+        "language-service",
```

Comment:
This needs to go into Azure/azure-sdk-tools/eng/common and pushed to all repos. You should not make local edits to anything in eng/common.